docs/pages/connect-your-client/tsh.mdx

```diff
@@ -586,7 +586,7 @@ $ tsh logout
 ## Copying files
 
 To securely copy files to and from cluster Nodes, use the `tsh scp` command. It
-is designed to mimic traditional `scp` as much as possible:
+is designed to mimic OpenSSH `scp` as much as possible:
```

Comment:
```suggestion
is designed to mimic OpenSSH's `scp` command as much as possible:
```

Reply:
I'd also add that from Teleport 10, we support SFTP and SCP protocols, so `scp` and `sftp` tools from the OpenSSH package should work.

Reply:
Done